optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/common/AbstractPersistable.java

```diff
@@ -39,11 +41,10 @@
     protected Integer tenantId;
 
     @Version
-    protected Long version;
```

Comment:
This needs to stay, we're using it for optimistic locking.

Reply:
`@Version` can be of type `OffsetDateTime`, and can still be used for optimistic locking (changed by hibernate on insert and update operation)

Reply:
Done